spec/system/partner_system_spec.rb

```diff
@@ -317,6 +317,21 @@
           end
         end
       end
+
+      context "when viewing prior distributions" do
```

Comment:
Since this test doesn't require any browser interactivity/javascript, it could probably be changed from a system spec to be a request spec. Because system specs are pretty slow.

Reply:
FWIW,  I guarantee @dorner will strongly suggest the same thing.

Reply:
Indeed. :grin: Can it please be moved to a request spec?